test-aether-integration-module/conf/pip/primary-requirements.txt

```diff
@@ -11,8 +11,7 @@
 
 aet.consumer
 aether.client
-aether.mocker
-aether.saladbar
+coreapi
```

Comment:
This dependency must be inherited from `aether-client`, it's ok having it here but you can also include the comment ;-)